app/models/notification.rb

```diff
@@ -41,4 +41,17 @@ def send_admin_email
     return unless admin_users
     NotificationMailer.send_admin_messages(self).deliver_later
   end
+
+  # attempt to not tread on MBIS application behaviour for now until
+  # notifications ana mail is fully decoupled
+  def project_user_ids(project_id)
+    return if project_id.nil?
+
+    application = Project.find(project_id)
+
+    users_associated_to_application = Project.find(project_id).users
```

Comment:
As above, with associations in place we could walk them e.g. `project.users`, which then gives access to scopes for filtering and `:pluck` - we could boil this down to something along the lines of `project.users.internal.pluck(:id)`.

I know we're not dealing with huge collections of users so it's not like we'll be hitting performance bottlenecks, so I feel like I'm splitting hairs a little.

Reply:
Yup - we don't have an internal scope but I'll add one. In the back of my mind this did make it PHE domain specific so maybe this scope would have to eventually be configurable